Code/JavaWrappers/RWMol.i

```diff
@@ -54,6 +54,8 @@
 %javaconst(1);
 #endif
 %include <GraphMol/FileParsers/FileParsers.h>
+%ignore RDKit::v2;
+%ignore RDKit::v2::SmilesParse;
```

Comment:
Swig 4.1 supports unique_ptr is this helps the wrappers move to v2

https://swig.org/Doc4.1/Library.html#Library_std_unique_ptr

Reply:
that's really good to know!